.github/workflows/assets.yaml

```diff
@@ -119,6 +119,46 @@ jobs:
     name: Kubernetes gRPC load test
     needs: build-and-push
     runs-on: ubuntu-latest
+    - name: Install Iter8
+      run: GOBIN=/usr/local/bin go install github.com/iter8-tools/iter8@v0.15
+    - name: Start kind cluster
+      uses: helm/kind-action@v1.5.0
+      with:
+        wait: 300s
+    - name: Create app
+      run: |
+        kubectl create deployment routeguide --image=golang --port=50051 \
+        -- bash -c "git clone -b v1.52.0 --depth 1 https://github.com/grpc/grpc-go; cd grpc-go/examples/route_guide; sed -i "''" "'"s/localhost//"'" server/server.go; go run server/server.go"
+        kubectl expose deployment routeguide --port=50051
+        kubectl wait --for=condition=available --timeout=60s deployment/routeguide
+
+    - name: Test gRPC service with grpcurl
+      run: |
+        curl -sO https://gist.githubusercontent.com/kalantar/510737f0fd58c0432a08e5b6e45ec97f/raw/524d6660284bf653ce0f29f3a25ed0e913c3df80/grpcurl-routeguide.yaml
+        kubectl apply -f grpcurl-routeguide.yaml
+        sleep 120
+        kubectl logs deploy/sleep
```

Comment:
I like having this step here to ensure that the gprc server is reachable.

Currently, the Kubernetes manifest is in a gist. Should this be moved into iter8?